src/Features/CSharp/Portable/CSharpFeaturesResources.Designer.cs

```diff
@@ -10,7 +10,6 @@
 
 namespace Microsoft.CodeAnalysis.CSharp {
     using System;
-    using System.Reflection;
```

Comment:
>using System.Reflection; [](start = 4, length = 24)

do you know what caused this change?
I'm wary of changes in generated files that cause flip-flops